sonar-csharp-plugin/pom.xml

```diff
@@ -171,66 +170,166 @@
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-enforcer-plugin</artifactId>
+            <version>1.4.1</version>
+            <executions>
+              <execution>
+                <id>enforce-property</id>
+                <goals>
+                  <goal>enforce</goal>
+                </goals>
+                <configuration>
+                  <rules>
+                    <requireProperty>
+                      <property>analyzer.version</property>
+                      <message>You must set the property analyzer.version to specify the version of the NuGet artifacts to download.</message>
+                      <regex>^(\d+)\.(\d+)\.(\d+)\.(\d+)$</regex>
+                      <regexMessage>The analyzer.version property must have 4 digits.</regexMessage>
+                    </requireProperty>
+                  </rules>
+                  <fail>true</fail>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>copy-plugin</id>
+                <phase>validate</phase>
+                <goals>
+                  <goal>copy</goal>
+                </goals>
+                <configuration>
+                  <artifactItems>
+                    <artifactItem>
+                      <groupId>${project.groupId}</groupId>
+                      <artifactId>SonarAnalyzer.CSharp</artifactId>
+                      <version>${analyzer.version}</version>
+                      <type>nupkg</type>
+                      <overWrite>true</overWrite>
+                    </artifactItem>
+                    <artifactItem>
+                      <groupId>${project.groupId}</groupId>
+                      <artifactId>SonarAnalyzer.RuleDocGenerator.CSharp</artifactId>
+                      <version>${analyzer.version}</version>
+                      <type>nupkg</type>
+                      <overWrite>true</overWrite>
+                    </artifactItem>
+                    <artifactItem>
+                      <groupId>${project.groupId}</groupId>
+                      <artifactId>SonarAnalyzer.Scanner</artifactId>
+                      <version>${analyzer.version}</version>
+                      <type>nupkg</type>
+                      <overWrite>true</overWrite>
+                    </artifactItem>
+                  </artifactItems>
+                  <outputDirectory>${sonarAnalyzer.workDirectory}</outputDirectory>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>1.7</version>
+            <executions>
+              <execution>
+                <id>copy-sqale-xml</id>
```

Comment:
The id of this execution doesn't seem to match what it does. Mainly because I don't see why `SonarAnalyzer.CSharp` and `SonarAnalyzer.Scanner` are unzipped, because I see only copying files from `SonarAnalyzer.RuleDocGenerator.CSharp`